GitReleaseManager.yaml

```diff
@@ -0,0 +1,23 @@
+create:
+  include-footer: false
+issue-labels-include:
+ - "t: Bug"
+ - "t: New Feature"
+ - "t: Improvement"
+ - "t: Task"
+issue-labels-exclude:
+ - "r: Rejected"
+ - "r: Replaced"
+issue-labels-alias:
+ - name:   "t: Bug"
+   header: Bug
+   plural: Bug
```

Comment:
Our release notes were not pluralizing the section, so  I have put a singular for plural.